config/buildspec_zero_code_change.yml

```diff
@@ -27,6 +27,7 @@ phases:
         - cd $CODEBUILD_SRC_DIR  && chmod +x config/protoc_downloader.sh && ./config/protoc_downloader.sh
         - pip install --upgrade pip==19.3.1
         - pip install -q pytest wheel pyYaml pytest-html pre-commit awscli
+        - pip install -q tensorflow_datasets
```

Comment:
```suggestion
        - pip install -q pytest wheel pyYaml pytest-html pre-commit awscli tensorflow_datasets
```

Reply:
Thought we'd separate infrastructure dependencies from code dependencies